test/functional/options/winfixbuf_spec.lua

```diff
@@ -8,7 +8,7 @@ describe("Nvim API calls with 'winfixbuf'", function()
     clear()
   end)
 
-  it("Calling vim.api.nvim_win_set_buf with 'winfixbuf'", function()
+  it("Calling vim.api.nvim_win_set_buf on other buffer with 'winfixbuf'", function()
```

Comment:
let's unverbose these test names. there is no need to mention 'winfixbuf' in the tests since they are in a `describe()` scope that already mentions 'winfixbuf'

```suggestion
  it("vim.api.nvim_win_set_buf on non-current buffer", function()
```